recipes/sdl_image/all/conanfile.py

```diff
@@ -96,17 +96,17 @@ def layout(self):
 
     def requirements(self):
         # Headers are exposed https://github.com/conan-io/conan-center-index/pull/16167#issuecomment-1508347351
-        self.requires("sdl/2.28.3", transitive_headers=True)
+        self.requires("sdl/2.28.5", transitive_headers=True)
```

Comment:
Maybe it would be better if we could use the latest sdl on ConanCenter, which is the 2.30.1? So the latest of ``sdl_image`` doesn't conflict with the latest of ``sdl``?

Reply:
Well, responding to myself, and checking other recipes in ConanCenter, the other recipes in ConanCenter depends on 2.28, so this cannot be done and would require some coordination, please be patience, this might take some time. We are trying to improve the CI to be able to do these changes more streamlined, but this is a big effort.